src/Controllers/login.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Request, Response } from 'express';
+import { checkLoginData } from 'Services/login.service';
+import { User } from 'Types/User';
+import jswonwebtoken from 'jsonwebtoken';
+
+export const loginController = async (
+    req: Request,
+    res: Response
+): Promise<void> => {
+    try {
+        const userData: User = req.body;
+
+        const user = await checkLoginData(userData);
+
+        if (user === null) {
+            res.sendStatus(401);
```

Comment:
I think that you could also send a custom message here so it can be displayed on the frontend app in a way we want

Reply:
resolved